### PR TITLE
Ronnie/postgress over igal

### PR DESCRIPTION
**Pull request by ronniel1**

### CODE REVIEW

File: cmd/main.go [hunk 1]
```diff
@@ -100,10 +100,10 @@ func main() {
 		kclient = nil
 	}
 
-	db, err := gorm.Open("mysql",
-		fmt.Sprintf("admin:admin@tcp(%s:%s)/installer?charset=utf8&parseTime=True&loc=Local",
+	// Connect to db
+	db, err := gorm.Open("postgres",
+		fmt.Sprintf("host=%s port=%s user=admin dbname=installer password=admin sslmode=disable",
```
Comment (filanov) [inline cmd/main.go:105]: if we already here maybe we should get the user db name and the password as parameters as well?

File: internal/common/common_unitest_db.go [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+	err := gDbCtx.pool.Purge(gDbCtx.resource)
+	Expect(err).ShouldNot(HaveOccurred())
+	gDbCtx.pool = nil
+}
+
+func PrepareTestDB(dbName string, extrasSchemas ...interface{}) *gorm.DB {
+	Expect(gDbCtx.pool).ShouldNot(BeNil())
+	Expect(gDbCtx.resource).ShouldNot(BeNil())
+	var dbTemp *gorm.DB
+	err := gDbCtx.pool.Retry(func() error {
```
Comment (filanov) [inline internal/common/common_unitest_db.go:47]: Is there a timeout to this retry?

**Comment by filanov** [comment]
@ronniel1 where can we see the logs of the DB? is there an option to set it to stdout?

File: internal/common/common_unitest_db.go (hunk 2 above)
Comment (ronniel1) [reply internal/common/common_unitest_db.go:47]: I think it is 15 minutes

Comment (filanov) [reply internal/common/common_unitest_db.go:47]: anyway default go test timeout is 10min. 
how long does it take to run all the tests this way?

File: cmd/main.go (hunk 1 above)
Comment (ronniel1) [reply cmd/main.go]: done

File: Makefile [hunk 3]
```diff
@@ -152,7 +152,10 @@ deploy-grafana: create-build-dir
 deploy-monitoring: deploy-olm deploy-prometheus deploy-grafana
 
 unit-test:
-	go test -v $(or ${TEST}, ${TEST}, $(shell go list ./... | grep -v subsystem)) -cover
+	docker stop postgres || true
+	docker run -d  --rm --name postgres -e POSTGRES_PASSWORD=admin -e POSTGRES_USER=admin -p 127.0.0.1:5432:5432 postgres:12.3-alpine -c 'max_connections=10000'
+	SKIP_UT_DB=1 go test -v $(or ${TEST}, ${TEST}, $(shell go list ./... | grep -v subsystem)) -cover
```
Comment (filanov) [inline Makefile:157]: if this part will return an error will the make process continue and kill postgres?

File: .drone.yml [hunk 4]
```diff
@@ -2,12 +2,10 @@ kind: pipeline
 name: default
 
 steps:
-  - name: mariadb
-    image: mariadb
+  - name: postgres
+    image: postgres:12.3-alpine
```
Comment (mhrivnak) [inline .drone.yml:6]: Downstream I think we'll want to use this: https://catalog.redhat.com/software/containers/rhel8/postgresql-12/5db133bd5a13461646df330b

Comment (mhrivnak) [reply .drone.yml:6]: And here is an upstream option: https://hub.docker.com/r/centos/postgresql-12-centos7

Comment (ronniel1) [reply .drone.yml:6]: done

File: Makefile (hunk 3 above)
Comment (ronniel1) [reply Makefile:157]: probably not will fix

**Pull request merged**